data/model/oci/tag.py

```diff
@@ -421,7 +421,7 @@ def retarget_tag(
         return created
 
 
-def delete_tag(repository_id, tag_name):
+def delete_tag(repository_id, tag_name, force=False):
```

Comment:
nit: `skip_recovery_period` or something similar instead of `force` might be clearer on what it does.

Reply:
Agreed, i changed the option name to `skipRecoveryPeriod`. I also changed the expiration to just outside the time machine window vs. setting it to 0 to prevent integrity errors. That change is [here.](https://github.com/quay/quay/pull/1751/files#diff-d54089ecfb2c9ed6312fb03ca1ea1cd1be9555c8dedf9138a2feac65ea530ff5R448-R461)